spec/machine/ARM/MachineOps.thy

```diff
@@ -140,6 +147,9 @@ and
   us_to_ticks_nonzero: "y \<noteq> 0 \<Longrightarrow> us_to_ticks y \<noteq> 0"
 and
   kernelWCET_ticks_no_overflow: "4 * unat (us_to_ticks (kernelWCET_us)) \<le> unat (max_word :: 64 word)"
+and
+  us_to_ticks_mult: "\<forall>n a. (unat n * unat (us_to_ticks a) \<le> unat (max_word :: 64 word))
+                           \<longrightarrow> n * us_to_ticks a = us_to_ticks (n * a)"
```

Comment:
I've had a quick look at this `us_to_ticks` function. In the interests of moving things forwards, I think that this is fine. If `us_to_ticks` is merely multiplying by a constant, this new axiom is satisfied. But it seems possible that if it is dividing by a constant, this axiom may not hold. In the future, we may need to rethink our approach here.

Reply:
We might need to defer this discussion for later. Maybe a JIRA issue would be good. Can you please do that, Michael?

Reply:
https://sel4.atlassian.net/browse/VER-1388